src/WebJobs.Extensions.DurableTask/Storage/AzureStorageAccountExplorer.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Azure.Data.Tables;
+using Azure.Storage.Blobs;
+using Azure.Storage.Queues;
+using DurableTask.AzureStorage;
+#if !FUNCTIONS_V1
+using Microsoft.Extensions.Azure;
+#endif
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Storage
+{
+    internal sealed class AzureStorageAccountExplorer : IAzureStorageAccountExplorer
```

Comment:
Let me know how you feel about any of these names lol. I needed to avoid calling this `AzureStorageClientProviderProvider`

Reply:
While re-reviewing this code, I was thinking of a similar name. The client providers use `IStorageServiceClientProvider`, but this interface essentially provides clients for the entire storage account. How about `IStorageAccountClientProvider`?

Reply:
Also added a more relevant `<inheritdoc/>` tag